.github/workflows/rust-ci.yml

```diff
@@ -26,7 +26,6 @@ jobs:
 
       - name: Build mvr-cli (debug)
         run: |
-          cd mvr-cli
```

Comment:
Wouldn't this fail as it needs to build from the `mvr-cli` folder?

Reply:
I moved the repo to a workspace, so we can use `cargo install --git`! Look at a comment I left on your PR too!